docs/enableexperimental.md

```diff
@@ -0,0 +1,37 @@
+# How to enable experimental features
+
+Fleet supports experimental features that are disabled by default and that can be enabled by the user.
+
+Enabling/disabling experimental features is done using extra environment variables that are available when deploying `rancher/fleet`.
+
+## Enabling an experimental feature
+
+At the moment we're writing this document `Fleet` has OCI storage as an experimental feature.
+
+### Enabling when installing Fleet stand-alone
+
+All you need to do is to pass something like:
+```
+--set-string extraEnv[0].name=EXPERIMENTAL_OCI_STORAGE \
+--set-string extraEnv[0].value=true \
+```
+to your helm install or update command. 
+
+Please note you have to use `--set-string` because otherwise the boolean value won't work as expected.
+
+### Enabling when installing Fleet with Rancher
+
+You can also activate the experimental features in `Fleet` when installing `Rancher`.
+
+The parameters are the same, but you have to add the `Fleet.` prefix.
```

Comment:
Would an uppercase "F" even be correct here? This looks like an accidental search-and-replace hit.
```suggestion
The parameters are the same, but you have to add the `fleet.` prefix.
```